src/tests/ftest/daos_test/daos_core_test.yaml

```diff
@@ -23,7 +23,7 @@ daos_tests:
     test_r:
       daos_test: r
       test_name: rebuild tests
-      args: -u subtests="0-1"
+      args: -u subtests="0-8"
```

Comment:
Why not all?

Reply:
there were failures on rebuild 10 when running on CI in VM, I am not able to figure out yet.

Reply:
Strange, the DTX patches have passed such test (on boro) with 7 servers